examples/sdk-interactive-examples/examples/example-attributefilter/README.md

```diff
@@ -0,0 +1,30 @@
+# GoodData.UI SDK - Interactive Examples - Attribute Filter Example
+
+This example demonstrates how to use the AttributeFilter component to filter data in a visualization.
+
+You can run this example in [CodeSandbox](https://codesandbox.io/p/sandbox/github/gooddata/gooddata-ui-sdk/tree/master/examples/sdk-interactive-examples/examples/example-attributefilter?file=/src/example/Example.tsx). Navigate [up](../../) for more information.
+
+[![Attribute Filter Example](./.example/preview.png)](https://codesandbox.io/p/sandbox/github/gooddata/gooddata-ui-sdk/tree/master/examples/sdk-interactive-examples/examples/example-attributefilter?file=/src/example/Example.tsx)
+
+## Run example locally
+
+### In sdk-ui-sdk 
+If you cloned whole sdk-ui-sdk repository
+
+1) ```rush install```
+2) ```rush build```
+3) navigate to example directory
+4) ```npm run start``` 
+
+### Just example directory
+
+1) navigate to example directory
```

Comment:
Is this safe? I would expect this could break the rush? 🤔

Reply:
Its not mean in rush (whole repo) ... but if you somehow just clone example dir  ...